test/addons/async-hooks-promise/test.js

```diff
@@ -26,7 +26,7 @@ hook0.disable();
 
 let pwrap = null;
 const hook1 = async_hooks.createHook({
-  init(id, type, tid, resource) {
+  init(_, __, ___, resource) {
```

Comment:
I agree with this, we do have the same code here

https://github.com/nodejs/node/blob/5bef2ccf20cceda7975f8bce860e0f60595482fc/test/known_issues/test-stream-writable-sync-error.js#L20-L22

Reply:
I would rather prefer to change that code then, to be honest… there’s nothing wrong with being explicit about what the function parameters are, and if nothing else, it reduces overhead when do you want to start using them

Reply:
This is also often added to explain that it is not being used and that the parameter has a placeholder, for example:  `init(_id, _type, _tid, resource) { ... }`